auraescript/README.md

```diff
@@ -1,62 +0,0 @@
-# AuraeScript
```

Comment:
is this not relevant any more?

Reply:
I put it back and made some updates to make it more inline with what we have.